src/EventManagement.Domain/Certificate.cs

```diff
@@ -9,32 +9,29 @@ public class Certificate {
         
         [Key]
         public int CertificateId { get; set; }
-
-        // Keys
-        //[DatabaseGenerated (DatabaseGeneratedOption.Identity)]
+        [DatabaseGenerated (DatabaseGeneratedOption.Identity)]
```

Comment:
This was not working the last time I tried which is why I commented it. Seems like I forgot to remove it before committing. Since the database wasn't setting an identity value, I had to manually set it using `Guid.NewGuid()`.